package.json

```diff
@@ -5,11 +5,11 @@
   "main": "app.js",
   "scripts": {
     "test": "jest --passWithNoTests",
-    "dev": "nodemon run app.js",
+    "dev": "nodemon src/app.ts",
     "lint": "eslint ./**/*.ts --fix",
     "build": "tsc --build",
     "clean": "tsc --build --clean",
-    "start": "tsc --build --clean && tsc --build && prettier --write src/ && node dist/app.js"
+    "start": "node dist/app.js"
```

Comment:
https://stackoverflow.com/a/47305304

I think this is best. It watches for .ts files change and recompile instantly, then recompile node afterwards. Just what we need.